.github/workflows/nightly.yml

```diff
@@ -55,8 +55,8 @@ jobs:
           - {os: ubuntu-latest, toolchain: stable,  target: x86_64-unknown-linux-musl, target_cpu: generic, cross: true}
           - {os: ubuntu-latest, toolchain: stable,  target: x86_64-unknown-linux-musl, target_cpu: broadwell, cross: true}
           - {os: ubuntu-latest, toolchain: stable, target: aarch64-linux-android, target_cpu: generic, cross: true}
-          - {os: ubuntu-latest, target: mips64el-unknown-linux-gnuabi64, cross: true}
-          - {os: ubuntu-latest, target: powerpc64le-unknown-linux-gnu, cross: true}
+          - {os: ubuntu-latest, toolchain: stable, target: mips64el-unknown-linux-gnuabi64, target_cpu: generic, cross: true}
```

Comment:
It might be more maintainable if we factor out `toolchain` from this table and add the few outliers with `include`:

```yaml
      matrix:
        config:
          # Linux
          - {os: ubuntu-latest, target: x86_64-unknown-linux-gnu, target_cpu: generic, cross: false}
          - {os: ubuntu-latest, target: x86_64-unknown-linux-gnu, target_cpu: broadwell, cross: false}
          # ...
        toolchain: stable
        include:
           # Windows
           - config: {os: windows-latest, target: x86_64-pc-windows-gnu, target_cpu: generic, cross: false}
             toolchain: stable-x86_64-pc-windows-gnu
           - config: {os: windows-latest, target: x86_64-pc-windows-msvc, target_cpu: generic, cross: false}
             toolchain: stable-x86_64-pc-windows-msvc
           # ...
```

Then, replace every occurrence of `${{ matrix.config.toolchain }}` with `${{ matrix.toolchain }}`.

Reply:
created an issue in #2310